bubbles/message_utils.py

```diff
@@ -1,19 +1,74 @@
-from typing import Any, Dict, Union
+from __future__ import annotations
 
+from typing import Any, Dict, Union, Callable, Optional, TYPE_CHECKING
+from slack_sdk.web.client import WebClient
 
-class MessageUtils:
-    def __init__(self, client, payload):
+if TYPE_CHECKING:
+    from bubbles.plugins import PluginManager
```

Comment:
What is this for?

Reply:
Type hinting without a circular dependency. Let's the class be imported for the IDE to parse, but won't get evaluated during runtime.